inference-engine/src/transformations/src/transformations/op_conversions/convert_interpolate1_to_interpolate4.cpp

```diff
@@ -15,55 +15,58 @@
 NGRAPH_RTTI_DEFINITION(ngraph::pass::ConvertInterpolate1ToInterpolate4, "ConvertInterpolate1ToInterpolate4", 0);
 
 ngraph::pass::ConvertInterpolate1ToInterpolate4::ConvertInterpolate1ToInterpolate4() {
-    auto interpolate1 = ngraph::pattern::wrap_type<ngraph::opset1::Interpolate>({pattern::any_input(pattern::has_static_rank()), pattern::any_input()});
+    auto interpolate1 = ngraph::pattern::wrap_type<ngraph::opset1::Interpolate>({pattern::any_input(), pattern::any_input()});
     ngraph::matcher_pass_callback callback = [this](pattern::Matcher& m) {
-        auto interpolate1 = std::dynamic_pointer_cast<ngraph::opset1::Interpolate>(m.get_match_root());
-        if (!interpolate1)
+        auto interpolationV0 = std::dynamic_pointer_cast<ngraph::op::v0::Interpolate>(m.get_match_root());
+        if (!interpolationV0) {
             return false;
+        }
+
+        auto& inp_shape = interpolationV0->get_input_shape(0);
```

Comment:
This method can throw an exception if operation has partial shape. Please first of all check that shapes are not dynamic.

Reply:
Fixed.